wallet/walletdb/interface_test.go

```diff
@@ -95,6 +95,30 @@ func testDeleteValues(tc *testContext, bucket walletdb.ReadWriteBucket, values m
 	return true
 }
 
+// testCursorDelete should remove all key/value pairs from the provided bucket.
+// It currently is expected to fail due to a reported bolt bug:
+// https://github.com/boltdb/bolt/issues/620
+func testCursorDelete(tc *testContext, bucket walletdb.ReadWriteBucket) bool {
```

Comment:
What is the purpose of this test? The comment says that it is expected to "fail", but the test only fails if the bug is not experienced.